independent-publisher-connectors/Secure Code Warrior/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7075 @@
+    "TeamEntry": {
+      "description": "This object represents team details",
+      "required": [
+        "name",
+        "status",
+        "timeSpent"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the team",
+          "type": "string",
+          "example": "Demo Team"
+        },
+        "status": {
+          "description": "Status of the team",
+          "type": "string",
+          "example": "enabled"
+        },
+        "timeSpent": {
+          "description": "Team time spent",
+          "type": "integer",
+          "example": 826765
+        }
+      }
+    },
+    "MostEngagedUsers": {
+      "description": "Most Engaged Users in the company",
+      "required": [
+        "users"
+      ],
+      "type": "object",
+      "properties": {
+        "users": {
+          "description": "User details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UserEntry"
+          }
+        },
+        "links": {
+          "description": "Related API pages and pagination details",
+          "type": "object",
+          "title": "Links",
+          "properties": {
+            "total_pages": {
+              "type": "integer",
+              "description": "The total number of pages in the results",
+              "example": 10
+            },
+            "results_per_page": {
+              "type": "integer",
+              "description": "The maximum number of results in each page",
+              "example": 100
+            },
+            "next_page": {
+              "type": "integer",
+              "description": "The page number of the next page (if one exists)",
+              "example": 3
+            },
+            "prev_page": {
+              "type": "integer",
+              "description": "The page number of the previous page (if one exists)",
+              "example": 1
+            },
+            "total_results": {
+              "type": "integer",
+              "description": "The total number of the results",
+              "example": 513
+            }
+          }
+        }
+      }
+    },
+    "UserEntry": {
+      "description": "This object represents user details",
+      "required": [
+        "name",
+        "status",
+        "timeSpent",
+        "resourceID"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the user",
+          "type": "string",
+          "example": "Demo User"
+        },
+        "status": {
+          "description": "Status of the user",
+          "type": "string",
+          "example": "enabled"
+        },
+        "timeSpent": {
+          "description": "Time spent by the user",
+          "type": "integer",
+          "example": 826765
+        },
+        "resourceID": {
+          "description": "resourceID of user",
+          "type": "string",
+          "example": "105a1bd6-c985-4a9f-abd8-9ac5cdf0c237"

```

Comment:
@troystaylor Please remove all the request ids from all the connector artifacts

Reply:
Done